graphql-server/src/schemas/custom.ts

```diff
@@ -1225,19 +1213,44 @@ function buildSamplesQueryBody({
     });
   }
 
+  // NOTE: For future reference, we can get a list of historical CMO labels and import dates by replacing
+  // '| sm.cmoSampleName' with '| {cmoSampleName: sm.cmoSampleName, importDate: sm.importDate}'
   const samplesQueryBody = `
     // Get Sample and the most recent SampleMetadata
     MATCH (s:Sample)-[:HAS_METADATA]->(sm:SampleMetadata)
-    WITH s, collect(sm) AS allSampleMetadata, max(sm.importDate) AS latestImportDate
-    WITH s, [sm IN allSampleMetadata WHERE sm.importDate = latestImportDate][0] AS latestSm
+    WITH
+      s,
+      collect(sm) AS allSampleMetadata,
+      max(sm.importDate) AS latestImportDate
+    WITH
+      s,
+      allSampleMetadata,
+      [sm IN allSampleMetadata WHERE sm.importDate = latestImportDate][0] AS latestSm
+
+    // Get a list of historical CMO sample labels
+    WITH
+      s,
+      latestSm,
+      apoc.text.join(
+        apoc.coll.toSet(
+          [sm IN apoc.coll.sortMaps(allSampleMetadata, "importDate")
+            WHERE sm.cmoSampleName <> latestSm.cmoSampleName AND sm.cmoSampleName <> ""
+            | sm.cmoSampleName + " (" + sm.importDate + ")"
+          ]
+        ),
+      ", ") AS historicalCmoSampleNames
```

Comment:
Only this part of the Cypher query body is new. The rest of the diffs is me adding indentation to be consistent with the other Cypher queries.

How to interpret this addition to the query:
```cypher
      // Combine the results into a comma-separated list
      apoc.text.join(
        // Convert list to set to remove duplicate SampleMetadatas with the same sample labels
        apoc.coll.toSet(
          // Filter a sorted list of SampleMetadatas only for the historical ones
          [sm IN apoc.coll.sortMaps(allSampleMetadata, "importDate")
            WHERE sm.cmoSampleName <> latestSm.cmoSampleName AND sm.cmoSampleName <> ""
            // For each result, put them in this format:
            | sm.cmoSampleName + " (" + sm.importDate + ")"
          ]
        ),
      ", ") AS historicalCmoSampleNames
```